python/arcticdb/version_store/_store.py

```diff
@@ -1735,13 +1744,21 @@ def compact_incomplete(
             Convert data to sparse format (for tick data only)
         metadata : `Optional[Any]`, default=None
             Add user-defined metadata in the same way as write etc
+        prune_previous_version
+            Removes the previous versions from the database after a successful compaction
+            (unless the previous version is in a snapshot).
```

Comment:
The docstring for the other methods that take this parameter is consistently:
`Removes previous (non-snapshotted) versions from the database.`

Reply:
I find the old documentation ambiguous

Reply:
Couldn't be bothered to update the other ones, so will use the old one to be consistent.